src/sql/SqlResults.h

```diff
@@ -96,6 +96,8 @@ class SqlResults : boost::noncopyable {
                               std::vector<std::string>&, std::vector<std::string>&, SqlErrorObject&);
     bool extractFirst4Columns(std::vector<std::string>&, std::vector<std::string>&, std::vector<std::string>&,
                               std::vector<std::string>&, SqlErrorObject&);
+    bool extractFirst5Columns(std::vector<std::string>&, std::vector<std::string>&, std::vector<std::string>&,
+                              std::vector<std::string>&, std::vector<std::string>&, SqlErrorObject&);
```

Comment:
I don't think this should be changed in this ticket, but this is odd. `bool extractFirstXColumns(int numOfCols, vector<vector<string>>&, SqlErrorObject&);` with `string& worker = vect[i][0];` `string& database = vect[i][1];`, etc.

Reply:
Do you mean missing parameter names in the method signatures, or something else? The following is perfectly legal syntax in C and C++:
```
void foo(T&)
```
However, I completely agree with you that it's a bit unusual in this context. I will fix a signature of this and similar methods of the class.

Reply:
The syntax is fine. It's just 5 different functions when one would do.

Reply:
Yep, I agree. I will introduce another method to fill up a 2D vector `vector<vector<string>>` as per your suggestion.